analyzer/main.zeek

```diff
@@ -295,7 +305,7 @@ event protocol_confirmation(c: connection, atype: Analyzer::Tag, aid: count) {
   }
 
 }
-
+@endif
```

Comment:
Could you update e.g., `tests/basic.zeek` to check that we emit no deprecation warnings.

You could e.g., adjust the invocation of `zeek`:

```diff
-# @TEST-EXEC: zeek -C -r ${TRACES}/ldap-simpleauth.pcap %INPUT
+# @TEST-EXEC: zeek -C -r ${TRACES}/ldap-simpleauth.pcap %INPUT >output 2>&1
+# @TEST-EXEC: btest-diff output
```

The baseline should be empty.